vm/devices/pci/pci_core/src/spec.rs

```diff
@@ -280,6 +280,21 @@ pub mod cfg_space {
         pub reserved2: u16,
     }
 
+    impl Command {
+        pub const VALID_BITS: u16 = Command::new()
```

Comment:
No. The whole point is that what's valid is a property of the _emulator_, not of the specification.

Reply:
I don't follow, what are you asking for?

Reply:
If someone changes `VALID_BITS` in this spec file, does it mean the emulator now supports the specified bit?

Reply:
I see. So move this const into the emu and rename it SUPPORTED_BITS?

Reply:
Yeah that seems reasonable.